src/secondary_thermal.cpp

```diff
@@ -19,6 +19,7 @@ void get_energy_index(
   f = 0.0;
   if (E >= energies.front()) {
     i = lower_bound_index(energies.begin(), energies.end(), E);
+    if (i+1 < energies.size())
```

Comment:
Looks like you need to run clang-format on this file

Reply:
clang-format did not run, because in my setup `tools/dec/post-commit.git-clang-format` was not set as executable. Fix that...